.github/workflows/benchmark.yaml

```
@@ -0,0 +1,49 @@
+        uses: actions/checkout@v4
+
+      - name: Setup Dart
+        uses: dart-lang/setup-dart@v1.3
+        with:
+          sdk: stable
+
+      - name: Cache Dart Dependencies
+        uses: actions/cache@v4
+        with:
+          path: ${{ env.PUB_CACHE_PATH }}
+          key: ${{ runner.os }}-pub-cache-${{ matrix.dart_sdk }}
```

actionlint (1.7.7)

property "dart_sdk" is not defined in object type {}

(expression)